iothub/service/src/MessageSystemPropertyNames.cs

```diff
@@ -8,40 +8,115 @@ namespace Microsoft.Azure.Devices
     /// </summary>
     public static class MessageSystemPropertyNames
     {
+        /// <summary>
+        /// A user-settable identifier for the messages. If this value is not supplied by the user,
+        /// the service client will set this to a new GUID only if you set <see cref="ServiceClientOptions.SdkAssignsMessageId"/>
+        /// property in <see cref="ServiceClientOptions"/>.
+        /// </summary>
         public const string MessageId = "message-id";
 
+        /// <summary>
+        /// Lock token of the received message.
+        /// A unique identifier for a cloud-to-device message used to complete, reject or abandon the message.
+        /// This value is provided to resolve race conditions when completing, rejecting, or abandoning messages.
+        /// <remarks>
+        /// If the lock token expires, the cloud-to-device message needs to be received again to complete, reject or abandon
+        /// the message without errors.
+        /// </remarks>
+        /// </summary>
```

Comment:
summary end tag should be above remarks. They aren't supposed to be nested, I believe.

Reply:
oops. will fix.